uiowa/uiowa
# [IWP] Adjust resource summary display in teaser

Resolves #7980 




# How to test


```
ddev blt ds --site=iwp.uiowa.edu
```
Add a few resources or sync from dev to local (`ddev drush sql-sync @iwp.dev @iwp.local`)
visit https://iwp.uiowa.ddev.site/media-gallery and see that the summaries are trimmed with an ellipses suffix
